### ISSUE TITLE
Consider using focus-visible on buttons/chips etc

### ISSUE THREAD

**Issue by IngSchei** [miljodir/md-components#272]

**Comment by aurorascharff** [comment]
"Implement :focus-visible for interactive elements where keyboard navigation should be clearly indicated, while keeping :focus where necessary for accessibility." hvilke steder ville dette vært? @IngSchei

**Comment by IngSchei** [comment]
@aurorascharff Godt spørsmål, men kanskje på inputfelt, f.eks?
